src/__tests__/config.test.ts

```diff
@@ -39,20 +39,15 @@ describe('validateConfiguration', () => {
   test('parses minimal configuration', () => {
     const data = { github: { owner: 'getsentry', repo: 'craft' } };
 
-    const projectConfig = validateConfiguration(data);
-
-    expect(projectConfig).toEqual(data);
+    expect(validateConfiguration(data)).toEqual(data);
   });
 
-  test('fails with empty configuration', () => {
-    // this ensures that we do actually run the expect line in the catch block
-    expect.assertions(1);
-
-    try {
-      validateConfiguration({});
-    } catch (e) {
-      expect(e.message).toMatch(/should have required property/i);
-    }
```

Comment:
Because the only required field was `github` in the config and with that also becoming optional it does **not** fail with empty config: the test case is now obsolete :)